package/MDAnalysis/core/topology.py

```diff
@@ -17,13 +17,58 @@
 Topology object --- :mod:`MDAnalysis.core.topology'
 ===================================================================
 
+
+"""
+
+"""
+# second docstring == multiline comment
+
+TODO Notes:
```

Comment:
OK then we can add a TODO for the issue-363 PR that we should convert all code TODO's introduced to issues on github.